docs/swag-test-commands.md

```
@@ -338,31 +338,31 @@ swag logs nginx-error 1001
 swag logs nginx-error -1
 ```
 
-## 9. CLEANUP_BACKUPS Action Commands
+## 9. BACKUPS Action Commands
 
 ### Basic Cleanup Commands
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)